Dockerfile

```diff
@@ -33,7 +36,8 @@ wget -q -O - https://repo.radeon.com/rocm/rocm.gpg.key | apt-key add - && \
 apt-get update && \
 DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     build-essential \
-    cmake \
+    cmake-data=3.15.1-0kitware1 \
+    cmake=3.15.1-0kitware1 \
```

Comment:
@JehandadKhan  I thought these lines are needed since base OS is Ubuntu 18.04 and we need more up-to-date cmake? Could you confirm?

Reply:
Indeed, these are part of the rbuild process, but since we could not build CK there due to CK's dependence on hipcc, I had to add them here as well. 

In light of [your update in CK](https://github.com/ROCmSoftwarePlatform/composable_kernel/tree/ck_clang_support_test) I propose we merge this PR as is and then fix the dependency issues in a separate PR.